tekton/release-cheat-sheet.md

```diff
@@ -1,121 +1,170 @@
+   The two URLs can be opened in the browser or via `curl` to download the release manifests.
 
-        ```bash
-        tkn tr logs -f create-draft-release-run-# this will end with a random string of characters
-        ```
+    1. The YAMLs are now released! Anyone installing Tekton Triggers will now get the new version. Time to create a new GitHub release announcement:
 
-    1. On successful completion, a URL will be logged. Visit that URL and sort the
-    release notes. **Double-check that the list of commits here matches your expectations
-    for the release.** You might need to remove incorrect commits or copy/paste commits
-    from the release branch. Refer to previous releases to confirm the expected format.
+    1. Create additional environment variables
 
-    1. Publish the GitHub release once all notes are correct and in order.
+    ```bash
+    TRIGGERS_OLD_VERSION=# Example: v0.11.1
+    TEKTON_PACKAGE=tektoncd/triggers
+    ```
 
-10. Edit `README.md` on `master` branch, add entry to docs table with latest release links.
+    1. Create a `PipelineResource` of type `git`
 
-11. Push & make PR for updated `README.md`
+    ```shell
+    cat <<EOF | kubectl --context dogfooding create -f -
+    apiVersion: tekton.dev/v1alpha1
+    kind: PipelineResource
+    metadata:
+      name: tekton-triggers-$(echo $TEKTON_VERSION | tr '.' '-')
+      namespace: default
+    spec:
+      type: git
+      params:
+        - name: url
+          value: 'https://github.com/tektoncd/triggers'
+        - name: revision
+          value: ${TRIGGERS_RELEASE_GIT_SHA}
+    EOF
+    ```
 
-12. **Important: Stop pointing `kubectl` at dogfooding cluster.**
+    1. Execute the Draft Release task.
 
     ```bash
-    kubectl config use-context my-dev-cluster
+    tkn --context dogfooding task start \
+      -i source="tekton-triggerss-$(echo $TEKTON_VERSION | tr '.' '-')" \
+      -i release-bucket=tekton-triggers-bucket \
+      -p package="${TEKTON_PACKAGE}" \
+      -p release-tag="${TRIGGERS_VERSION}" \
+      -p previous-release-tag="${TRIGGERS_OLD_VERSION}" \
+      create-draft-release
     ```
 
-13. Test release that you just made.
+    1. Watch logs of create-draft-release
+
+    1. On successful completion, a URL will be logged. Visit that URL and look through the release notes.
+      1. Manually add upgrade and deprecation notices based on the generated release notes
+      1. Double-check that the list of commits here matches your expectations
+         for the release. You might need to remove incorrect commits or copy/paste commits
+         from the release branch. Refer to previous releases to confirm the expected format.
+
+    1. Un-check the "This is a pre-release" checkbox since you're making a legit for-reals release!
+
+    1. Publish the GitHub release once all notes are correct and in order.
+
+1. Edit `README.md` on `master` branch, add entry to docs table with latest release links.
+
+1. Push & make PR for updated `README.md`
+
```

Comment:
One minor thing: I'd like to keep the step where we explicitly mention the release captain stop pointing to the dogfooding cluster so that we don't accidentally update it with the new release (step 12. previously)

Reply:
done 👍